lib/resty/radixtree.lua

```diff
@@ -54,6 +54,7 @@ local ngx_re      = require("ngx.re")
 local empty_table = {}
 local str_find    = string.find
 local str_lower   = string.lower
+local table_remove = table.remove
```

Comment:
Please rename it to `move_tab` to make the naming consistent.

Reply:
It's table.remove, not table.move.

Reply:
Yes, typo. Please rename it to `remove_tab`.